htlcswitch/switch.go

```diff
@@ -790,22 +790,6 @@ func (s *Switch) handleLocalDispatch(pkt *htlcPacket) error {
 			}
 		}
 
-		if link.Bandwidth() < htlc.Amount {
-			err := fmt.Errorf("Link %v has insufficient capacity: "+
-				"need %v, has %v", pkt.outgoingChanID,
-				htlc.Amount, link.Bandwidth())
-			log.Error(err)
-
-			// The update does not need to be populated as the error
-			// will be returned back to the router.
-			htlcErr := lnwire.NewTemporaryChannelFailure(nil)
-			return &ForwardingError{
```

Comment:
The behavior here has changed: as is we send a `ForwardingError` back to the router, while this changes it to be a naked onion error.

Reply:
This case is now checked in `CheckHtlcTransit` and that error is also wrapped in a `ForwardingError` above.